spring-data-jdbc/src/main/java/org/springframework/data/jdbc/core/convert/Jsr310TimestampBasedConverters.java

```diff
@@ -57,7 +58,7 @@ public abstract class Jsr310TimestampBasedConverters {
 	 */
 	public static Collection<Converter<?, ?>> getConvertersToRegister() {
 
-		List<Converter<?, ?>> converters = new ArrayList<>(8);
+		List<Converter<?, ?>> converters = new ArrayList<>(28);
```

Comment:
Why 28?

Reply:
Meh, that was a leftover from the code below (`TODO`). We should discuss that one.